### ISSUE TITLE
RFE: Keyboard navigation

### ISSUE THREAD

**Issue by henrypijames** [webtorrent/webtorrent-desktop#1101]
WTD has very little support for keyboard navigation so far. For example, in the confirmation dialog for removing torrents, you cannot hit Enter for confirming the default Remove button. This is related to Issue #970, but covers other (including future) dialogs. It would be nice to traverse the entire list of torrents with the keyboard.

**Comment by feross** [comment]
We have lots of shortcuts already – look in the menu bar to see what shortcuts are available. But we don't support navigating the torrent list, starting a torrent from the keyboard, etc. Let's do it.

**Comment by gkiely** [comment]
Could we change these shortcuts to match youtube?
